NS_Install2.sh

```diff
@@ -18,7 +20,7 @@ fi
 if [ ! -s /usr/local/etc/no-ip2.conf ]
 then
 cd /usr/src
-sudo tar -xzf /srv/nightscout-vps/helper/noip-duc-linux.tar.gz
+sudo tar -xzf /srv/"$GIT_REPOSITRY"/helper/noip-duc-linux.tar.gz
```

Comment:
The untarring will go away.   We will allow the user to install the latest Nightscout directly from the official Nightscout repository.